### PR TITLE
Simplify animation

### PR DESCRIPTION
**Pull request by ceenell**
This modifies the map animation so that is based on the 5 binned categories rather than the continuous percentile data. This reduces the amount of animation in some places, and increases it in others. There's a flashiness happening at sites that go rapidly between different categories, where in other places the sites are more consistent looking allowing you to notice what's changing.

To switch this I pulled out the path animation function from the `animateGWL` function. `animatePathD` is called using `.each` because `.each` allows us to build the animation uniquely for each element. I thought this would make it possible to include logic as whether or not to run the function if the data changes (`var today` and `var yesterday`). BUT to really simplify things, this logic needs to come before the transition itself is created somehow.

I'm adding this to a new branch to be able to put on beta and look at page performance vs previous version. I think this should be better because there is less path interpolation when the data are binned, but could eliminate >80% of `transition` calls if is only run when data are changed between days.

### CODE REVIEW

**Comment by ceenell** [comment]
Doing this also eliminated the console error messages that were occurring when a site had NA for data for some reason. Not sure if that means the sites are staying on the map when they are NA or if they turn off. Another good thing with this is that the round shape for the normal category is used, making those sites more apparent.

**Comment by ceenell** [comment]
Animation code updated approach using `.each` that only call transition if data change based on feedback from @hcorson-dosch

File: src/components/GWL.vue [hunk 1]
```diff
@@ -382,6 +356,13 @@ export default {
           .domain([-40, -25, 25, 40])
           .range(this.pal_BuBr) 
 
+        // set scale for path shape
+        var quant_path = [...new Set(this.quant_peaks.map(function(d) { return d.path_quant}))];
```
Comment (hcorson-dosch) [inline src/components/GWL.vue:360]: 👀 that `Set()` function is good to know about!

**Review by hcorson-dosch — APPROVED** [review]
Nice discovery of `.each` and naming transitions, @ceenell ! The animation is feeling pretty darn snappy now!!

**Pull request merged**